src/main/java/gregtech/common/metatileentities/electric/multiblockpart/MetaTileEntityFluidHatch.java

```diff
@@ -101,13 +102,21 @@ private int getInventorySize() {
     @Override
     protected void initializeInventory() {
         super.initializeInventory();
-        FluidTank fluidTank = new FluidTank(getInventorySize());
+        FluidTank fluidTank = new NotifiableFluidTank(getInventorySize(),getController(),isExportHatch);
         this.fluidTankHandler = new FluidTankList(false, fluidTank);
         this.exportFluids = fluidTankHandler;
         this.importFluids = isExportHatch ? new FluidTankList(false) : fluidTankHandler;
         this.fluidInventory = fluidTankHandler;
     }
 
+    @Override
+    public void addEntityToDirtyableHandlers(MetaTileEntity metaTileEntity) {
+        NotifiableFluidTank handler;
+        handler = (NotifiableFluidTank) getExportFluids().getFluidTanks().get(0);
```

Comment:
There should be if with `instanceof NotifiableFluidTank ` as it can be inherited and `initializeInventory` overrided